File: BUOI1/BAI_2/app/src/main/java/com/example/bai_2/MainActivity.java
package com.example.bai_2;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.EditText;
import android.widget.TextView;

import org.w3c.dom.Text;

public class MainActivity extends AppCompatActivity {
    EditText edta, edtb;
    int result;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.bai7);
    }

    private void HandleSum(){
        edta=(EditText) findViewById(R.id.soa);
        int a= Integer.parseInt(edta.getText()+"");
        edtb=(EditText) findViewById(R.id.sob);
        int b = Integer.parseInt(edtb.getText()+"");
        result = a + b;
        TextView t_result = (TextView) findViewById(R.id.text_result);
        t_result.setText(result);
    }

}

File: BUOI1/BAI10/app/src/main/java/com/example/bai10/MainActivity.kt
package com.example.bai10

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.TextView
import java.lang.Math.pow

abstract class MainActivity : AppCompatActivity(){
//    var a: Double = 0.0
//    var b: Double = 0.0
//    var c: Int = 0
//    val tResult:TextView = findViewById(R.id.textResult)
//    val myClickListener = View.OnClickListener {View->
//        when(View.id){
//            R.id.so9->a = 0.0
////            R.id.so8->
////            R.id.so7->
////            R.id.so6->
////            R.id.so5->
////            R.id.so4->
////            R.id.so3->a
////            R.id.so2->a
////            R.id.so1->a
////            R.id.so0->a
//        }
//    }
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

//    public fun handleSo(a: Double, c:Int): Double{
//        var result:Double
//        result = a*pow(10.0, c.toDouble())
//        var resultP: Double = result
//        tResult.text = resultP.toInt().toString()
//        return result
//    }
}